data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template

```diff
@@ -23,10 +23,16 @@ for name in ironic-api ironic-conductor ironic-inspector dnsmasq httpd mariadb i
 done
 
 # Start the provisioning nic if not already started
-PROVISIONING_NIC=ens4
-if ! nmcli -t device | grep "$PROVISIONING_NIC:ethernet:connected:provisioning"; then
+PROVISIONING_NIC={{.PlatformData.BareMetal.ProvisioningInterface}}
```

Comment:
PlatformData.BareMetal.ProvisioningInterface is the nic we get from the install-config, right?  In which case it may well be different to the desired value for the bootstrap VM (where a hardcoded value is probably reasonable, given that the installer defines that VM?)